src/amo/pages/AddonVersions/index.js

```diff
@@ -175,6 +175,7 @@ export class AddonVersionsBase extends React.Component<InternalProps> {
                   headerText={i18n.gettext('Latest version')}
                   key="latestVersion"
                   version={latestVersion}
+                  showLinkInsteadOfButton={false}
```

Comment:
We shouldn't be specifying `false` for a boolean property, instead, we should just omit the property. I notice if we do that that Flow complains, so the definition of the `showLinkInsteadOfButton` property in `AddonVersionCard` will have to change to optional.